config/controller.yaml

```diff
@@ -56,8 +56,10 @@ spec:
 
           # These images are pulled from Dockerhub, by digest, as of April 15, 2020.
           "-nop-image", "tianon/true@sha256:009cce421096698832595ce039aa13fa44327d96beedb84282a69d3dbcf5a81b",
-          "-shell-image", "busybox@sha256:a2490cec4484ee6c1068ba3a05f89934010c85242f736280b35343483b2264b6",
           "-gsutil-image", "google/cloud-sdk@sha256:6e8676464c7581b2dc824956b112a61c95e4144642bec035e6db38e3384cae2e",
+          # The shell image must be root in order to create directories and copy files to PVCs.
```

Comment:
Is that needed ? the PVC should be owned by the user running the image?

Reply:
I guess it doesn't on Kubernetes but only on OpenShift...

Reply:
Pretty sure this blew up in Prow, so whatever's in the e2e test hit this.